spec/openapi.yaml

```diff
@@ -963,6 +980,91 @@ components:
           type: string
           example: "/dev/tty"
 
+    StatusJob:
+      type: object
+      description: Telemetry info about current job, all values are optional
+      properties:
+        id:
+          type: integer
+          example: 42
+        progress:
+          type: number
+          example: 0.42
+        time_remaining:
+          type: integer
+          example: 520
+        time_printing:
+          type: integer
+          example: 526
+
+    StatusPrinter:
+      type: object
+      description: Telemetry info about printer, all values except state are optional
+      properties:
+        state:
+          type: string
+          enum: [ IDLE, BUSY, PRINTING, PAUSED, FINISHED, STOPPED, ERROR, ATTTENTION, READY ]
+        temp_nozzle:
```

Comment:
Will there target temperatures? Nozzle Size?

Reply:
Nozzle size is part of api/v1/info, I can add target temperatures